packages/sql-utils/src/index.ts

```diff
@@ -164,7 +164,7 @@ export class Builder {
 
     // query shorthand
     if (Array.isArray(query)) {
-      conditions.push(this.createMemberQuery(key, query))
+      conditions.push(this.createEqualQuery(key, query.join(',')))
```

Comment:
The changes here do not match the expected behavior. Is it the original code that caused the issue?